src/components/NotFound.svelte

```diff
@@ -3,21 +3,39 @@
   import { twemoji } from "@app/lib/utils";
 
   import Button from "@app/components/Button.svelte";
-  import Modal from "@app/components/Modal.svelte";
 
-  export let title = "";
-  export let subtitle = "";
+  export let title: string;
+  export let subtitle: string;
```

Comment:
I made them required on purpose, so that we always supply both props. Otherwise this component wouldn't look right.

<img width="649" alt="Screenshot 2023-01-24 at 09 46 54" src="https://user-images.githubusercontent.com/158411/214247661-cd57fff4-ea01-4272-8ab8-5715c52f0c83.png">

In what situation do you imagine we would omit one of these props?

Reply:
Hmm no you're right 👍 discard this comment